src/App.js

```diff
@@ -400,6 +467,8 @@ const AppStack = () => {
         />
         <Stack.Screen name='PaymentRequestDetail' component={PaymentRequestDetail} />
         <Stack.Screen name='RegisterToken' component={RegisterTokenStack} />
+        <Stack.Screen name='RegisterNanoContract' component={RegisterNanoContractStack} />
+        <Stack.Screen name='Register' component={RegisterTokenStack} />
```

Comment:
Why do you need this? The token stack is already above

Reply:
I must have forgotten it there.